src/js/components/Button/doc.js

```diff
@@ -47,7 +47,7 @@ export const doc = Button => {
       )
       .defaultValue(false),
     focusIndicator: PropTypes.bool
-      .description("Whether when 'plain' it should receive a focus outline.")
+      .description('Whether it should receive a focus outline.')
```

Comment:
I think this is only true for `plain` button.

Reply:
Hmm you're right I must have tested on the wrong page. So it looks like Select is the only one whose docs need updating. Will update PR

Reply:
`  ${props =>
    props.focus && (!props.plain || props.focusIndicator) && focusStyle()}`